### PR TITLE
61 top level groups of categories not listed

### PR DESCRIPTION
**Pull request by astowny**
Fixed the displayed data next to the donut chart in BEGESv5 mode, and changed a little bit the colors

### CODE REVIEW

File: src/utils/CategoryColors.ts
Comment (lepeuvedic-natixar) [inline src/utils/CategoryColors.ts:1]: In the context of #5 please divide this function in 2 distinct functions: ``getColorByEra`` and ``getColorByCategory``.
"1" and "operation" should be the same color. "2" could be "upstream" but clearly "3" should NOT be "downstream" since it is unrelated.

File: src/components/natixarComponents/ScopeTable/NatixarExpandableRow.tsx
Comment (lepeuvedic-natixar) [inline src/components/natixarComponents/ScopeTable/NatixarExpandableRow.tsx:1]: Much better! :smiley:

File: src/components/natixarComponents/CO2DonutSection/EmissionByScopeDonutSection.tsx
Comment (lepeuvedic-natixar) [inline src/components/natixarComponents/CO2DonutSection/EmissionByScopeDonutSection.tsx:1]: Need to disjoin processing of categories and eras in the future. They are unrelated concepts.

**Review by lepeuvedic-natixar — APPROVED** [review]
Further work to be done in #5

**Review by lepeuvedic-natixar — DISMISSED** [review]
OK

**Review by lepeuvedic — APPROVED** [review]
Approved

**Review by lepeuvedic — APPROVED** [review]
Good

**Comment by lepeuvedic** [comment]
Just merge  main into branch #61

**Comment by lepeuvedic** [comment]
If you include anything that's not been included already I'll have to reject the change, do a git reset --hard to truncate the branch history, etc.   You should never, ever merge from any branch other than main or dev (and it's much safer to merge from dev only when dev is not broken).

File: src/app-layout/Header/HeaderContent/MobileSection.tsx
Comment (lepeuvedic) [inline src/app-layout/Header/HeaderContent/MobileSection.tsx:1]: Unrelated.

File: src/app-layout/Header/SubHeaderContent/EmissionFilterMenu.tsx
Comment (lepeuvedic) [inline src/app-layout/Header/SubHeaderContent/EmissionFilterMenu.tsx:1]: Unrelated (related to vanishing filters)!

File: src/app-layout/Header/SubHeaderContent/index.tsx
Comment (lepeuvedic) [inline src/app-layout/Header/SubHeaderContent/index.tsx:1]: Unrelated

File: src/app-layout/Header/index.tsx
Comment (lepeuvedic) [inline src/app-layout/Header/index.tsx:1]: Unrelated

**Review by lepeuvedic — CHANGES_REQUESTED** [review]
Remove last-minute unrelated edits.
